_posts/en/meetings/2016-12-22-meeting.md

```diff
@@ -0,0 +1,125 @@
+---
+title: IRC meeting summary for 2016-12-22
+permalink: /en/meetings/2016/12/22/
+name: 2016-12-22-meeting
+type: meetings
+layout: page
+lang: en

```

Comment:
Yeah, I don't know what it is. It's not in the source file.

It's also on all the other meetings, I assume something new from github. Don't know what it means or how to remove it though.